### DISCUSSION TITLE
javascript:// added to URLblocklist, and I'm kinda scared rip

### DISCUSSION THREAD

**Discussion by Huntlb** [3kh0/ext-remover#100] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Title, and philipbruh's answer doesn't work, I still can't run any bookmarklets on the page. I also had to edit the code so it went to chrome.google.com/webstore_, because webstorex is added to the URLblocklist as well.

**Comment by Huntlb** [comment]
Wait actually when you create the update can you also make a non-bookmarklet version that you can just put into console? That would be super helpful for like the bookmarklets being disabled.

**Reply by Coolkid560** [reply]
Yeah, But Some Schools Block That Too.

**Reply by LogA1Cyber** [reply]
how do you get around that block?

**Comment by Coolkid560** [comment]
I Mean before the update

**Comment by 3kh0** [comment, marked as answer]
Duplicate #38
